src/modules/message-parsers.js

```diff
@@ -6,7 +8,11 @@
 module.exports.in = (msg) => {
   // if sender put a json header, we parse it to avoid the pain for the consumer
   if (msg.properties.contentType === 'application/json') {
-    return JSON.parse(msg.content.toString());
+    const content = JSON.parse(msg.content.toString());
```

Comment:
maybe oyu can adda check to see if content is set on message so toString() does not fail. Basically like here https://github.com/dial-once/node-bunnymq/pull/85/files#diff-e94fc0818279d2f3c707ffab934db749R33